ui/apps/platform/cypress/integration/systemConfig/systemConfig.test.js

```diff
@@ -65,13 +65,17 @@ describe('System Configuration', () => {
     });
 
     it('should not render Edit button if READ_ACCESS to resource', () => {
-        const staticResponseForPermissions = {
-            fixture: 'auth/mypermissionsMinimalAccess.json',
-        };
+        cy.fixture('auth/mypermissionsMinimalAccess.json').then(({ resourceToAccess }) => {
+            const staticResponseForPermissions = {
+                body: {
+                    resourceToAccess: { ...resourceToAccess, Administration: 'READ_ACCESS' },
+                },
+            };
 
-        visitSystemConfigurationWithStaticResponseForPermissions(staticResponseForPermissions);
+            visitSystemConfigurationWithStaticResponseForPermissions(staticResponseForPermissions);
```

Comment:
GitHub diff seems to show about a tab's worth of rogue whitespace on this line, yet the PR passed style checks?

Is our FE style check broken?

Reply:
Good eyes again.

The test failed because the fixture had `NO_ACCESS` instead of `READ_ACCESS` and now the page is not rendered. Indentation level increased because of `cy.fixture` above.